_episodes/13-scheduler.md

```diff
@@ -135,12 +131,13 @@ Let's illustrate this by example. By default, a job's name is the name of the sc
 Submit the following job (`{{ site.sched_submit }} {{ site.sched_submit_options }} example-job.sh`):
 
 ```
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
`-l` flag (run like a login shell) is extraneous, since none of the user's environment is relevant to the script.